### PR TITLE
adding elmercurio.com/Inversiones

### PR DESCRIPTION
**Pull request by dalacost**
adding support for elmercurio.com/Inversiones

### CODE REVIEW

**Comment by iamadamdev** [comment]
Also update README.md and manifest-ff.json

File: README.md [hunk 1]
```diff
@@ -48,6 +48,7 @@
 [Denver Post](https://www.denverpost.com)\
 [Dynamed Plus](https://www.dynamed.com)\
 [Eindhovens Dagblad](https://www.ed.nl)\
+[El Mercurio Inversiones](https://www.elmercurio.com/Inversiones)\
```
Comment (iamadamdev) [inline README.md:51]: Should this bypass work for all elmercurio.com sections?

Comment (dalacost) [reply README.md:51]: for now, elmercurio.com is using paywall for the "Inversiones" section only. I read this portal every day, so if they add some extra paywall i will update this.

Comment (chagoguila) [reply README.md:51]: I believe that /inversiones and plain .com are different.
For /inversiones disabling javascript will allow to read without problems

Comment (dalacost) [reply README.md:51]: > Should this bypass work for all elmercurio.com sections?

added support.

**Comment by iamadamdev** [comment]
Thanks for your contribution

**Pull request merged**